pkg/gitproviders/common.go

```diff
@@ -449,6 +481,23 @@ func (h defaultGitProviderHandler) CreatePullRequestToOrgRepo(orgRepRef gitprovi
 	return pr, nil
 }
 
+func (h defaultGitProviderHandler) OauthConfig() OauthProviderConfig {
```

Comment:
Can you add a bit of clarification on these two functions?  They are off an object, defaultGitProviderHander, but don't use the object in the implementation.  Having comments will help us when we return to look at this code.

Reply:
The `defaultGitProviderHandler` needs to match the `GitProviderHandler` interface, so that is why the function users a receiver.